games-emulation/melonds/patches/melonds-0.9.5.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
```

Comment:
This one could be a good candidate for upstream +1

Reply:
Yes, this is the patch that I'll upstream after the merge of this PR :)